### ISSUE TITLE
Fail faster when connection lost

### ISSUE THREAD

**Issue by aaronshaf** [scrimba/community#242]
... otherwise one realizes very late and has to refresh after making a lot of changes :)

**Comment by somebee** [comment]
Yeah, it's very bad behaviour. The client knows perfectly well that the server is not accepting the changes, but we don't tell anywhere in the ui. Will fix 👍
